resources/checks/A002_pgversion.sh

```diff
@@ -0,0 +1,8 @@
+# Collect postgres version info
+ssh ${HOST} "${_PSQL} -f - " <<SQL
+select
+    json_build_object('version', version(),
+        'server_version_num', current_setting('server_version_num'),
+        'server_major_ver', SPLIT_PART(current_setting('server_version'), '.', 1),
```

Comment:
this will work with 10+, but will give wrong result for 9.6 and older versions:
```
postila_ru=# select SPLIT_PART(current_setting('server_version'), '.', 1), version();
 split_part |                                                                     version
------------+-------------------------------------------------------------------------------------------------------------------------------------------------
 9          | PostgreSQL 9.6.11 on x86_64-pc-linux-gnu (Ubuntu 9.6.11-1.pgdg16.04+1), compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609, 64-bit
(1 row)
```

-- major version here is `9.6`

Reply:
Remove these values? Or it values useful and need get correct value?

Reply:
@dmius let's use this:
```
tetst=# select regexp_replace(current_setting('server_version'), e'\\.\\d+$', '') as major_version;
 major_version
---------------
 9.6
(1 row)

test=# select regexp_replace(current_setting('server_version'), e'^.*\\.', '') as minor_version;
 minor_version
---------------
 11
(1 row)
```
/cc @Nastradamus

Reply:
ок

Reply:
First way is not work
```postila_ru=# select regexp_replace(version(), e'\\.\\d+$', '') as major_version;
                                                        major_version
-----------------------------------------------------------------------------------------------------------------------------
 PostgreSQL 9.6.11 on x86_64-pc-linux-gnu (Debian 9.6.11-1.pgdg80+1), compiled by gcc (Debian 4.9.2-10+deb8u1) 4.9.2, 64-bit
(1 row)
```
```postila_ru=# select regexp_replace(current_setting('server_version'), e'^.*\\.', '') as minor_version;
 minor_version
---------------
 11
(1 row)
```